### DISCUSSION TITLE
Changing default header and text fonts in Pluto

### DISCUSSION THREAD

**Discussion by rpherman** [fonsp/Pluto.jl#1060] [Questions about using Pluto] [answered]
I did not see where I could change the default font in starting a Pluto notebook, or in an existing notebook.
I am guessing I can edit the HTML/CSS of the produced page, or there is a file that specifies styles and fonts, but I am not familiar with front end enough to easily do this. Thanks.

**Comment by fonsp** [comment, marked as answer]
You can add a cell like this:

```julia
html"""
<style>
pluto-output {
    font-family: monospace;
    color: green;
}
pluto-output h1 {
    font-family: "Comis Sans MS", cursive;
    color: red;
}
</style>
"""
```

See all the default styles here: https://github.com/fonsp/Pluto.jl/blob/bac224bf655a30d7b9ccaa9d66a53c23f0f33012/frontend/editor.css#L81-L380

**Reply by fonsp** [reply]
To change the styles locally, you can fork Pluto
